components/campaign/CampaignTable.js

```diff
@@ -1,15 +1,51 @@
 import React from 'react'
-import Link from '../Link'
 import { sortBy, prop } from 'ramda'
-import { formatDecimal } from '../../lib/utils/format'
 import CSVExport from '../../components/CSVExport'
-import TableHeaders from '../common/TableHeaders'
-import { tableHeaderNames } from '../../lib/enums'
+import Table from '../common/Table'
+
+const tableSchema = {
```

Comment:
Moving forward, do you think we should keep the tableSchema on the component level? If we think that is the way to go ( something I'm leaning towards), would it make sense to include `columnOrder` and `displayTooltip` attributes on the headers object?

Reply:
I was thinking that displayTooltip could be either:
1. as an array, it could be less repetitive than adding a true false to each header
2. as an object, it makes sense to be a property on each header

I couldn't figure how the column order could be collapsed as a property on the headers? would it be a numeric ranking? I think an array makes sense because it has an order (the index of the element)

Reply:
If the schema is defined on the component level we could leave the order of the objects as the implicit order of the headers, otherwise I was thinking a numeric value for the order. 

With this new setup, I think it makes sense to add the tooltip as a property on the table header object